user_tools/src/spark_rapids_tools/tools/qualx/model.py

```diff
@@ -101,28 +103,43 @@ def train(
     y_tune = cpu_aug_tbl.loc[cpu_aug_tbl['split'] != 'test', label_col]
     dtune = xgb.DMatrix(X_tune, y_tune)
 
-    best_params = tune_hyperparameters(X_tune, y_tune, n_trials)
-    logger.info(best_params)
-
-    # train model w/ best hyperparameters using data splits
-    base_params = {
-        'random_state': 0,
-        'objective': 'reg:squarederror',
-        'eval_metric': ['mae', 'mape'],  # applied to eval_set/test_data if provided
-        'booster': 'gbtree',
-    }
-    xgb_params = {**base_params, **best_params}
-    xgb_params.pop('n_estimators')
+    if base_model:
+        # use hyper-parameters from base model (w/ modifications to learning rate and num trees)
+        xgb_params = {}
+        cfg = json.loads(base_model.save_config())
+        train_params = cfg['learner']['gradient_booster']['tree_train_param']
+        xgb_params['eta'] = float(train_params['eta']) / 10.0     # decrease learning rate
```

Comment:
Why is the learning rate decreased (for my own education at least)?

Reply:
These adjustments are somewhat arbitrary (based on internet readings along with local experiments).  There probably can be more work done to pick better values, but these provided a decent balance between learning the new data vs. forgetting the original data.

Note that these values might need to be adjusted per dataset (i.e. "fine-tuning" may require some data-science/hand-holding).  Note: one alternative would be to just load the `save_config()` files as-is, and tell the user modify those JSON files directly if needed.

I tried to find some references... these were the best I could come up with:
- https://xgboost.readthedocs.io/en/stable/python/examples/continuation.html
- https://www.baeldung.com/cs/fine-tuning-nn#:~:text=That%20implies%20that%20the%20starting,new%20dataset%20in%20small%20steps. (for fine-tuning neural networks).

Reply:
We probably want these values available via a config as opposed to hard-coded.  That can be a follow-on though as it's low priority.